saw-core/src/SAWCore/Term/Certified.hs

```diff
@@ -166,6 +227,19 @@ scApply sc f arg =
      ctx <- unifyContexts "scApply" (rawCtx f) (rawCtx arg)
      pure (Term ctx tm tp)
 
+-- possible errors: not a pi type, bad argument type, context mismatch
```

Comment:
This can also throw a "Not a subtype" error, right?

(Similar comments apply to `scAscribe`.)